index.html

```diff
@@ -183,16 +193,17 @@
       </tbody>
     </table>
 
-    <p class="note">The <code>gAMA</code> and embedded ICC profile are provided solely for compatibility with processors that do
-    not conform to this specification.</p>
+    <p class="note">The <code>gAMA</code> chunk, <code>cHRM</code> chunk, and embedded ICC profile specified in the table above are
+    not necessarily accurate, and, even if accurate, their processing is not required to reproduce the image as intended by the
+    author. They are provided for graceful fallback for implementations that do not recognize the semantics associated with a
```

Comment:
> You end up with an SDR image with a peak luminance of 80 cd/m^2,, right?

Well SDR systems typically scale peak sRGB luminance to their peak luminance, so I would expect the same to apply here.